content/tutorial-build-run-rippled-ubuntu.md

```diff
@@ -89,7 +89,7 @@ These instructions use Ubuntu's APT (Advanced Packaging Tool) to install the sof
 
 10. Build `rippled` binary executable from source code. This may take about 30 minutes, depending on your hardware specs.
 
-        scons
+        scons ***TODO: If 1.66.0, I get errors. Expect that the errors will go away after Scott D's PR is applied to rippled***
```

Comment:
Tip: You can specify `-j <number of jobs>` here too. I think the space between the j and the number of jobs is probably optional? Not sure.

For what it's worth, on my PC, building rippled 0.90.0-b5 with 4 jobs cut the build time down to just 5 minutes.

Reply:
I ran ./b2 -j 4 -- with the space between -j and 4. The job finished with the space included, so I added it in the doc. However, the time it took didn’t get shorter for me. I’m wonder if it is something about it running on AWS?

I ran scons with a -j value twice -- and it hung at the same location each time. When I removed the -j it ran as usual. I left the -j value out of the doc, just because I wasn’t able to get it to work in my workflow -- though I do trust that it should work and that it worked for you!

Reply:
Generally, you'll only see an improvement with higher values of <j> when you have at least <j> CPU cores on your server. It's quite possible your test AWS instance only has one or two CPU cores, so you wouldn't see an improvement when using -j 4.

Reply:
You'd also need some relatively significant amounts of RAM to compile rippled, maybe your instance ran out of memory?

Reply:
Alternatively you can compile in nounity mode, but that'll take a bit longer and doesn't seem to be well tested.